packages/sst/src/constructs/NextjsSite.ts

```diff
@@ -271,6 +278,64 @@ export class NextjsSite extends SsrSite {
     server?.addEnvironment("REVALIDATION_QUEUE_URL", queue.queueUrl);
     server?.addEnvironment("REVALIDATION_QUEUE_REGION", Stack.of(this).region);
     queue.grantSendMessages(server?.role!);
+
+    const app = this.node.root as App;
+
+    const table = new Table(this, "RevalidationTable", {
+      partitionKey: { name: "tag", type: AttributeType.STRING },
+      sortKey: { name: "path", type: AttributeType.STRING },
+      pointInTimeRecovery: true,
+      billingMode: BillingMode.PAY_PER_REQUEST,
+      tableName: app.logicalPrefixedName("next-revalidation-table"),
+    });
+
+    table.addGlobalSecondaryIndex({
+      indexName: "revalidate",
+      partitionKey: { name: "path", type: AttributeType.STRING },
+      sortKey: { name: "revalidatedAt", type: AttributeType.NUMBER },
+    });
+
+    server?.addEnvironment("CACHE_DYNAMO_TABLE", table.tableName);
+    table.grantReadWriteData(server?.role!);
+
+    const dynamodbProviderPath = path.join(this.props.path, ".open-next", "dynamodb-provider");
+
+    if(fs.existsSync(dynamodbProviderPath)) {
```

Comment:
Should the table created above be moved inside the `if` block? That is, we don't want to create the DDB Table if the app does not store any cache.

Reply:
What's cheaper, storing the caches in DDB or in S3?

Reply:
S3 much cheaper. But we can't look up by cache if store in S3, can we?

Reply:
S3 is indeed cheaper, but there is no good way to make tags revalidation works within lambda. 
We could use the same method that they use on standalone of a single json files responsible for all the tags, but it will just work really really bad on lambda.

Reply:
I recall discussion EFS. Google has articles stating that it doesn't add coldstarts, but some people say it does... The downside is that it has to be in a VPC and doesn't work on the edge.